system/system_monitor/launch/system_monitor.launch.xml

```diff
@@ -1,12 +1,12 @@
 <launch>
-  <arg name="cpu_monitor_config_file" default="$(find-pkg-share system_monitor)/config/cpu_monitor.param.yaml"/>
-  <arg name="hdd_monitor_config_file" default="$(find-pkg-share system_monitor)/config/hdd_monitor.param.yaml"/>
-  <arg name="mem_monitor_config_file" default="$(find-pkg-share system_monitor)/config/mem_monitor.param.yaml"/>
-  <arg name="net_monitor_config_file" default="$(find-pkg-share system_monitor)/config/net_monitor.param.yaml"/>
-  <arg name="ntp_monitor_config_file" default="$(find-pkg-share system_monitor)/config/ntp_monitor.param.yaml"/>
-  <arg name="process_monitor_config_file" default="$(find-pkg-share system_monitor)/config/process_monitor.param.yaml"/>
-  <arg name="gpu_monitor_config_file" default="$(find-pkg-share system_monitor)/config/gpu_monitor.param.yaml"/>
-  <arg name="voltage_monitor_config_file" default="$(find-pkg-share system_monitor)/config/voltage_monitor.param.yaml"/>
+  <arg name="cpu_monitor_config_file" default="$(find-pkg-share tier4_system_launch)/config/system_monitor/cpu_monitor.param.yaml"/>
+  <arg name="hdd_monitor_config_file" default="$(find-pkg-share tier4_system_launch)/config/system_monitor/hdd_monitor.param.yaml"/>
+  <arg name="mem_monitor_config_file" default="$(find-pkg-share tier4_system_launch)/config/system_monitor/mem_monitor.param.yaml"/>
+  <arg name="net_monitor_config_file" default="$(find-pkg-share tier4_system_launch)/config/system_monitor/net_monitor.param.yaml"/>
+  <arg name="ntp_monitor_config_file" default="$(find-pkg-share tier4_system_launch)/config/system_monitor/ntp_monitor.param.yaml"/>
+  <arg name="process_monitor_config_file" default="$(find-pkg-share tier4_system_launch)/config/system_monitor/process_monitor.param.yaml"/>
+  <arg name="gpu_monitor_config_file" default="$(find-pkg-share tier4_system_launch)/config/system_monitor/gpu_monitor.param.yaml"/>
+  <arg name="voltage_monitor_config_file" default="$(find-pkg-share tier4_system_launch)/config/system_monitor/voltage_monitor.param.yaml"/>
```

Comment:
@HansRobo @takayuki5168 Is this okay? Not typo fixes.

Reply:
Not okay

Reply:
Reverted in https://github.com/autowarefoundation/autoware.universe/pull/2216/commits/1ac191a1c26238094a8913aa41aa3cf31eac21a1

Reply:
Could you check if scenario simulator (or planning simulator) works well with this PR in order to make sure that this kind of mistakes do not exist.